staging/src/k8s.io/component-base/config/types.go

```diff
@@ -210,3 +210,16 @@ func (l *VerbosityLevel) Set(value string) error {
 func (l *VerbosityLevel) Type() string {
 	return "Level"
 }
+
+// TracingConfiguration provides versioned configuration for OpenTelemetry tracing clients.
+type TracingConfiguration struct {
+	// Endpoint of the collector this component will report traces to.
+	// The syntax is defined in https://github.com/grpc/grpc/blob/master/doc/naming.md.
+	// Recommended default is the otlp grpc default, localhost:4317
+	// The connection is insecure, and does not currently support TLS.
+	Endpoint *string
+
+	// SamplingRatePerMillion is the number of samples to collect per million spans.
+	// Defaults to 0.
```

Comment:
```suggestion
	// +featureGate=KubeletTracing
	// +optional
```

This defaults to nil/unset so I don't think this is accurate?